src/main/java/spoon/reflect/visitor/CtBiScannerDefault.java

```diff
@@ -37,15 +29,15 @@
  */
 // autogenerated by CtBiScannerGenerator
 public class CtBiScannerDefault extends spoon.reflect.visitor.CtAbstractBiScanner {
-	protected Deque<CtElement> stack = new ArrayDeque<>();
+	protected java.util.Deque<spoon.reflect.declaration.CtElement> stack = new java.util.ArrayDeque<>();
```

Comment:
Is it correct that current generator produces fully qualified names here?

Reply:
yes, the BiScanner generator does not use autoimport